src/TFMiniPlus.cpp

```diff
@@ -87,33 +85,37 @@ bool TFMiniPlus::setFrameRate(uint16_t framerate) {
   uint8_t newRateLow = (uint8_t)framerate;
   uint8_t buffer[6] = {CMD_FRAME_MARKER, 0x06, 0x03, newRateLow, newRateHight};
   uint8_t commandResponse[6];
-  buffer[6] = TFMiniPlus::generateChecksum(buffer, 5);
```

Comment:
You had a buffer overrun here. buffer[6] is 1 byte beyond the end of buffer. That's why it's better to use buffer size when accessing it instead of hard coding indexes.